Dockerfile

```diff
@@ -54,7 +54,7 @@ ENV \
     GO_DEP_VERSION=0.5.4 \
     GO_VERSION=1.16.5 \
     HASKELL_STACK_VERSION=2.1.3 \
-    NPM_VERSION=6.14.2 \
+    NPM_VERSION=7.20.3 \
```

Comment:
Why not update to the latest version 7.20.6?

Reply:
Commit message:
* It's a bit confusing that you mention node in the title but NPM in the body.
* Could you shortly explain why it fixes the linked issue? Because it's not apparent from the Stackoverflow thread how that's related to the node version.
* Could you link the changelog / release notes as we usually do?